SIHRDmodel with variability.Rmd

```diff
@@ -0,0 +1,260 @@
+---
+title: "SIHRD Model"
+author: "Linh Tang, Britney He, Bowen Mince"
+date: "12/11/2020"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(dplyr)
+library(ggplot2)
+```
+
+```{r}
+model.data = function(N, beta, I.0, p, epsilon, theta, gamma, TimeH, TimeNH){
+  I.rate = beta/N
+  psi = 1/TimeH
+  lambda = 1/TimeNH
+  curr.day = 0
+  S = p * N - I.0
+  NS.0 = N - S - I.0
+  NS = NS.0
+  NewInfected = I.0
+  Total.Hospital = 0
+  Total.NonHospital = 0
+  Total.Infected = NewInfected + Total.Hospital + Total.NonHospital
+  Total.Infective = NewInfected + Total.NonHospital
+  Hospital.Recovered = 0
+  NonHospital.Recovered = 0
+  Hospital.Dead = 0
+  NonHospital.Dead = 0
+  Total.Recovered = Hospital.Recovered + NonHospital.Recovered
+  Total.Dead = Hospital.Dead + NonHospital.Dead
+  Pop = S + NS + Total.Infected + Total.Dead
+  Leaving.Hospital = 0
+  Leaving.NonHospital = 0
+  Recovered.Hospital.today = 0
+  Recovered.NonHospital.today = 0
+  Entering.Hospital = 0
+  
+  # If you want a full table like the excel, uncomment all lines in the df below
+  df = data_frame(Day = curr.day,
+                  Susceptible = S,
+                  Non.Susceptible = NS,
+                  Daily.New.Case = NewInfected,
+                  Total.In.Hospital = Total.Hospital,
+                  # Total.NonHospital = Total.NonHospital,
+                  Infected = Total.Infected,
+                  # Total.Infective = Total.Infective,
+                  # Recovered.In.Hospital = Hospital.Recovered,
+                  # Dead.In.Hospital = Hospital.Dead,
+                  Total.Recovered = Total.Recovered,
+                  Total.Dead = Total.Dead)
+  
+  # Watch the disease until it ends (daily new case = 0) or at max 200 days
+  while(NewInfected * S > 0 & curr.day < 200){
+    curr.day = curr.day + 1
+    
+    # Number of individuals recovered from the infected in hospital (previous day)
+    Leaving.Hospital = rbinom(1, Total.Hospital, psi)
+    Recovered.Hospital.today <- rbinom(1, Leaving.Hospital, theta)

```

Comment:
Can you change <- to = just for consistency? Sorry for being nit-pick but it is also easier to detect on Github lol